System.Net.TestData/ContosoCertificateGeneration/testservereku.contoso.com.inf

```diff
@@ -0,0 +1,17 @@
+; Export password: testcertificate
```

Comment:
Yes, this is intentional and a side-effect of having the certificates generated using powershell, without an INF or though our CA.